### PR TITLE
Updating powerd plugin components

### PR DESCRIPTION
**Pull request by g2pop**
This change brings the powerd plugin up to date with local changes

### CODE REVIEW

File: plugins/powerd/meson.build [hunk 1]
```diff
@@ -0,0 +1,26 @@
+if host_machine.system() == 'linux'
+  cargs = ['-DG_LOG_DOMAIN="FuPluginPowerd"']
+
+shared_module('fu_plugin_cros_powerd',
```
Comment (hughsie) [inline plugins/powerd/meson.build:4]: i think either the module needs to be called fu_plugin_powerd or the directory needs to be cros-powerd. I guess it depends on whether powerd is CrOS specific or not.

File: plugins/powerd/fu-plugin-powerd.c [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	GDBusProxy		*proxy;		/* nullable */
+};
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
+        
+        return;
```
Comment (hughsie) [inline plugins/powerd/fu-plugin-powerd.c:23]: you don't need a return from void

File: plugins/powerd/meson.build (hunk 1 above)
Comment (g2pop) [reply plugins/powerd/meson.build:4]: That makes sense. I think we've settled on just fu-plugin-powerd.

File: plugins/powerd/fu-plugin-powerd.c (hunk 2 above)
Comment (g2pop) [reply plugins/powerd/fu-plugin-powerd.c:23]: Thank you, I went back and adjusted this.

Comment (hughsie) [reply plugins/powerd/fu-plugin-powerd.c:23]: If it helps, do you want me to make you a committer on the main repo, and then you can push a wip/g2pop/powerd branch? if it's WIP we can scribble all over it and then when it's ready for review we can squash it into a sensible commit and merge.

Comment (g2pop) [reply plugins/powerd/fu-plugin-powerd.c:23]: Thank you, right now though, we still have more foundational work to do. I think we'll try to make the new plugin functional while on a fork, and then push it to a branch on the main repo for you to review at a higher level.

Comment (hughsie) [reply plugins/powerd/fu-plugin-powerd.c:23]: Okay, sounds like a plan. Yell if you need any help or advice.

**Pull request closed without merging**